### ISSUE TITLE
The initial position of the VSCode is shifted down when the native titlebar is on

### ISSUE THREAD

**Issue by popov895** [microsoft/vscode#117180]
- VS Code Version: 1.53.2
- OS Version: GNOME 3.38.4 X11 (Fedora 33)

The initial position of the VSCode is shifted down with every start of application when the native titlebar is on. Please, watch [this video](https://streamable.com/2tswdk).

Steps to Reproduce:

1. Launch VSCode and move its window to the required position
2. In the settings, select the native view for the window titlebar (Window > Title Bar Style > native)
2. Close VSCode
3.  Launch VSCode again

Does this issue occur when all extensions are disabled?: Yes

**Comment by bpasero** [comment]
So it does not reproduce with custom titlebar?

**Comment by popov895** [comment]
Yes, only with a native title bar. There is another issue with the custom title bar: the VSCode window grows a few pixels on each launch. I don't know if they are related or not.

**Comment by deepak1556** [comment]
Did the issue start only with `1.53` ?

**Comment by deepak1556** [comment]
I was able to repro with the native titlebar setup described in the video, I am attaching the window states on open and shutdown for 4 different launches https://gist.github.com/deepak1556/2d4b1e8d15281508e50e2c8a43e3c9f1 . Based on the log the `y` coordinate increases by `37px` before shutdown until the app can no longer go beyond the visible workspace.

Also I could not repro this issue with native titlebar using electron-fiddle.

**Comment by bpasero** [comment]
Since I am asking Electron for bounds I suspect an upstream issue:

https://github.com/microsoft/vscode/blob/ae0a103755b381fb5c1641b0c6789c936688c978/src/vs/platform/windows/electron-main/window.ts#L1140-L1145

**Comment by popov895** [comment]
Same on Wayland

**Comment by popov895** [comment]
Any news?

**Comment by popov895** [comment]
Still no progress on this issue?

**Comment by popov895** [comment]
Looks like it has been fixed for now.

**Comment by popov895** [comment]
> Looks like it has been fixed for now.

Nope, I was wrong:(

**Comment by deepak1556** [comment]
@VictorOF please check if the issue repro with our exploration builds https://github.com/microsoft/vscode/issues/166265#issuecomment-1397224038 , if it does then the next step would be to create a minimal repro with https://www.electronjs.org/fiddle

**Comment by deepak1556** [comment]
@VictorOF if you still feel like working on this, I would suggest to file a bug with your gist at https://github.com/electron/electron as next step.
